src/components/badges/UnreadBadge.tsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
```

Comment:
I am 4/5 we should not treat the UnreadBadge as a different component.
We could just use the `MentionBadge` with `0` or `null` as unreadCount to determine the state.

@deanwhillier WDYT?

Reply:
Yesss Michel I was thinking this today too, when I was texting you my thoughts :P but I couldn't remember if we were dead set on having it as two components. IMO it should be one too, like you said 0 or shape = circle